Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Plural forms in artifact origins localization

### Changelog:
- Added plural forms in artifact origins localization (e.g. `this ring`, but `these bracers`)


### Developer changelog:
- Changed how `target_renamed_artifact` is saved


### How to test:

Proofread [this spreadsheet](https://docs.google.com/spreadsheets/d/1FVVfKRCmTzPfiqoKvcfllqOhR613hpCzm2XJ8Gk1moU/edit#gid=1339305026&range=1:25), make sure that every custom localization (`GetArtifact[Singular/Plural]`) is correct and origins are eligible for both singular and plural artifacts (e.g. `necklace and its power`, but `bracers and their power`)

Character finder -> type `bracers` -> search all -> open character treasury -> check artifacts descriptions
Character finder -> type `necklace` -> search all -> open character treasury -> check artifacts descriptions
